bbn-test-4/covenant-committee/README.md

```diff
@@ -0,0 +1,38 @@
+# Covenant Committee
+
+The Covenant Committee comprises entities operating
+[Covenant Signer](https://github.com/babylonchain/covenant-signer) daemons.
+
+For this testnet, the Committee Committee has 9 members, 3 of which are operated
+by the Babylon Foundation. A full list of the endpoints follows below, grouped
+by the operating entity.
```

Comment:
Let's state that the quorum and current members can be found in the parameters and link there?

Reply:
Added